addons/calendar/models/calendar_recurrence.py

```diff
@@ -142,19 +142,24 @@ def _compute_name(self):
             else:
                 end = ''
 
-            if recurrence.rrule_type == 'weeky':
+            if recurrence.rrule_type == 'weekly':
```

Comment:
This typo should be fixed in stable.

Reply:
I will create another PR :-). It's been there since December of 2019 and I think it is only used in a few mail templates.
Note: the rest of the changes is needed too. `weekday_fields = (self._fields[weekday_to_field(w)] for w in weekdays)` will crash if executed.